src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -0,0 +1,2201 @@
+	}
+
+	private JRadioButton getAllRadioButton() {
+		if (radioButtonAll == null) {
+			radioButtonAll = new JRadioButton(MessageBundle.getMessage("angal.common.all.btn"));
+			radioButtonAll.setSelected(true);
+		}
+		return radioButtonAll;
+	}
+
+	private JRadioButton getMedicalWithNonZeroQuantityRadioButton() {
+		if (radioOnlyNonZero == null) {
+			radioOnlyNonZero = new JRadioButton(MessageBundle.getMessage("angal.inventory.medicalwithonlynonzeroqty.btn"));
+		}
+		return radioOnlyNonZero;
+	}
+
+	private JRadioButton getMedicalWithMovementRadioButton() {
+		if (radioWithMovement == null) {
+			radioWithMovement = new JRadioButton(MessageBundle.getMessage("angal.inventory.medicalwithmovementonly.btn"));
+		}
+		return radioWithMovement;
+	}
+
+	private JButton getCancelButton() {
+		if (jButtonCancel == null) {
+			jButtonCancel = new JButton(MessageBundle.getMessage("angal.common.cancel.btn"));
+			jButtonCancel.setMnemonic(MessageBundle.getMnemonic("angal.common.cancel.btn.key"));
+		}
+		jButtonCancel.addActionListener(actionEvent -> frame.dispose());
+		return jButtonCancel;
+	}
+
+	private List<MedicalInventoryRow> loadNewInventoryTable(boolean withNonZeroQty, MedicalType medicalTypeSelected) throws OHServiceException {
+		List<MedicalInventoryRow> inventoryRowsList = getMedicalInventoryRows(null);
+		if (withNonZeroQty) {
+			inventoryRowsList = inventoryRowsList.stream().filter(inv -> inv.getTheoreticQty() > 0).collect(Collectors.toList());
+		}
+		if (medicalTypeSelected != null) {
+			inventoryRowsList = inventoryRowsList.stream()
+				.filter(inv -> inv.getMedical().getType().getDescription().equals(medicalTypeSelected.getDescription()))
+				.collect(Collectors.toList());
+		}
+		return inventoryRowsList;
+	}
+
+	private List<MedicalInventoryRow> loadNewInventoryTable(MedicalType medicalTypeSelected) throws OHServiceException {
+		List<MedicalInventoryRow> inventoryRowsList = getMedicalInventoryRowsWithMovement();
+		if (medicalTypeSelected != null) {
+			inventoryRowsList = inventoryRowsList.stream()
+				.filter(inv -> inv.getMedical().getType().getDescription().equals(medicalTypeSelected.getDescription()))
+				.collect(Collectors.toList());
+		}
+		return inventoryRowsList;
+	}
+
+	private JButton getOkButton() {
+		if (jButtonOk == null) {
+			jButtonOk = new JButton(MessageBundle.getMessage("angal.common.ok.btn"));
+			jButtonOk.setMnemonic(MessageBundle.getMnemonic("angal.common.ok.btn.key"));
+			jButtonOk.addActionListener(actionEvent -> {
+				actions.forEach((key, value) -> {
+					if (key.isSelected()) {
+						actions.get(key).run();
+					}
+				});
+				jButtonCancel.doClick();
+			});
+		}
+		return jButtonOk;
+	}
+
+	private boolean areAllMedicalsInInventory() throws OHServiceException {
+		Set<Medical> inventorySet = new HashSet<>();
+		for (MedicalInventoryRow row : inventoryRowSearchList) {
+			inventorySet.add(row.getMedical());
+		}
+		return medicals.size() == inventorySet.size() ? true : false;
```

Comment:
remove the unnecessary and redundant boolean literals; the comparison itself returns true and false.

Reply:
Thanks!